GUI.py

```diff
@@ -125,6 +137,10 @@ def create_full():
     if controller_type == "Switch":
         controller_id = "Switch"
     elif controller_type == "Steam Deck":
+        if button_layout not in STEAMDECK_LAYOUTS:
+            error = UnsupportedCombinationError(controller_type, button_layout)
+            print(error.get_message())
```

Comment:
added this print so that it shows up in the GUI output log and not just the terminal

Reply:
Hey! I appreciate you making this change. Steam Controller and Switch Controller also only support Normal layouts, so its the three of them with this issue -- however I plan on making a version that would allow further customization (it will open a menu for the controller customization, defaulting to switch and containing an image of the current layout). The logic in this new window would only allow supported customizations, as it will generate drop boxes based off of the controller type you set. I will HOPEFULLY get this out by the weekend-- I have been swamped with work recently.

Reply:
Sounds great and definitely is more thought out vs. just a patch.